searcher.py

```diff
@@ -30,9 +31,12 @@ def search(self, client, output_path_guest, topic_path_host, topic_path_guest, g
             os.path.abspath(topic_path_host): {
                 "bind": topic_path_guest,
                 "mode": "ro"
-            },
+            }
         }
 
+        if len(self.config.test_split) > 0:
+            volumes[self.config.test_split] = {"bind": test_split_path_guest, "mode": "ro"}
```

Comment:
Should this added to `search_args` to be passed to the container?

Reply:
Yes! I added this arg on the run.py module. If the user passes a test set of query ids to perform the search on, then these lines mount the files with the query ids in the docker.